pkgs/by-name/to/toml-cli/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, fetchCrate, rustPlatform, testers, toml-cli }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "toml-cli";
+  version = "0.2.3";
+
+  src = fetchCrate {
+    inherit pname version;
```

```suggestion
    inherit version;
    pname = "toml-cli";
```
Allow overriding pname.